homeassistant/components/sonos/number.py

```diff
@@ -30,6 +31,23 @@
 _LOGGER = logging.getLogger(__name__)
 
 
+def _balance_to_number(state: tuple[int, int]) -> float:
+    left = state[0]  # 0..100
+    right = state[1]
+    # (x, 0) -> -100; (x, x) -> 0; (0, x) -> 100
```

Comment:
I'm not sure this comment is very obvious. Perhaps you could expand on how this works in a docstring on this function?